modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -143,37 +145,45 @@ class ONNXCompiled {
     void run();
 };
 
-static void appendExecutionProvider(Ort::SessionOptions          *session_options,
-                                    const cv::gapi::onnx::ep::EP &execution_provider) {
+static void addOpenVINOExecutionProvider(Ort::SessionOptions *session_options,
+                                         const cv::gapi::onnx::ep::OpenVINO &ov_ep) {
+     OrtOpenVINOProviderOptions options;
+     options.device_type = ov_ep.device_type.c_str();
+     options.cache_dir = ov_ep.cache_dir.c_str();
+     options.num_of_threads = ov_ep.num_of_threads;
+     options.enable_opencl_throttling = ov_ep.enable_opencl_throttling;
+     options.enable_dynamic_shapes = ov_ep.enable_dynamic_shapes;
+     options.context = nullptr;
+
+     try {
+        session_options->AppendExecutionProvider_OpenVINO(options);
+     } catch (const std::exception &e) {
+         std::stringstream ss;
+         ss << "ONNX Backend: Failed to enable OpenVINO Execution Provider: "
+            << e.what() << "\nMake sure that onnxruntime has"
+                           " been compiled with OpenVINO support.";
```

Comment:
again, ONNX Runtime on the local machine may be different from the ONNX Runtime used to build OpenCV..

Reply:
For `OpenVINO EP` it doesn't matter `AppendExecutionProvider_OpenVINO` will search for corresponding dll in runtime.
I'd remove this message about compilation with OpenVINO support...

Reply:
Done